src/Okta.Sdk.IntegrationTests/AuthorizationServersScenarios.cs

```diff
@@ -0,0 +1,879 @@
+            }
+            finally
+            {
+                await createdPolicy.DeactivateAsync();
+                await testClient.AuthorizationServers.DeleteAuthorizationServerPolicyAsync(createdPolicy.Id, createdPolicy.Id);
+                await createdAuthorizationServer.DeactivateAsync();
+                await testClient.AuthorizationServers.DeleteAuthorizationServerAsync(createdAuthorizationServer.Id);
+            }
+        }
+
+        [Fact(Skip = "OKTA-303121: Api validation failed: com.saasure.framework.exception.KeyStoreLimitExceededException (400, E0000001): Unable to create new authorization server. You cannot create authorization servers because signing keys could not be generated. To fix this issue, contact support.")]
+        public async Task GetAuthorizationServerPolicy()
+        {
+            var testClient = TestClient.Create();
+            var testAuthorizationServerName = $"{SdkPrefix}:Test AuthZ Server ({TestClient.RandomString(4)})";
+
+            var testAuthorizationServer = new AuthorizationServer
+            {
+                Name = testAuthorizationServerName,
+                Description = "Test Authorization Server",
+                Audiences = new string[] { "api://default" },
+            };
+            var testPolicy = new Policy
+            {
+                Type = PolicyType.OauthAuthorizationPolicy,
+                Status = "ACTIVE",
+                Name = "Test Policy",
+                Description = "Test policy",
+                Priority = 1,
+                Conditions = new PolicyRuleConditions
+                {
+                    Clients = new ClientPolicyCondition
+                    {
+                        Include = new List<string> { "ALL_CLIENTS" },
+                    },
+                },
+            };
+
+            var createdAuthorizationServer = await testClient.AuthorizationServers.CreateAuthorizationServerAsync(testAuthorizationServer);
+            var createdPolicy = await testClient.AuthorizationServers.CreateAuthorizationServerPolicyAsync(testPolicy, createdAuthorizationServer.Id);
+            try
+            {
+                createdAuthorizationServer.Should().NotBeNull();
+                createdPolicy.Should().NotBeNull();
+
+                var retrievedPolicy = await testClient.AuthorizationServers.GetAuthorizationServerPolicyAsync(createdAuthorizationServer.Id, createdPolicy.Id);
+                retrievedPolicy.Should().NotBeNull();
+                retrievedPolicy.Id.Should().Be(createdPolicy.Id);
+                retrievedPolicy.Name.Should().Be(createdPolicy.Name);
+                retrievedPolicy.Description.Should().Be(createdPolicy.Description);
+            }
+            finally
+            {
+                await createdPolicy.DeactivateAsync();
+                await testClient.AuthorizationServers.DeleteAuthorizationServerPolicyAsync(createdPolicy.Id, createdPolicy.Id);
```

Comment:
Same comment here. Policy id twice?

Reply:
Oops! Copy and pasted my mistake over and over again ☹️ .  I'll fix it.

Reply:
I suggest running these tests against your dev org to make sure they are working fine.

Reply:
Good idea! 👍